### PR TITLE
Fix: Issue #901 Add auth0 environment variables for integrations test

### PR DESCRIPTION
**Pull request by m3talsmith**
Given a new contributor, with a clean development environment, when you first run tests following the [CONTRIBUTING](https://github.com/auth0/auth0-cli/blob/main/CONTRIBUTING.md) document, you end up with no running integration tests. 

### 🔧 Changes

This PR addresses the technical and documentation problem in issue #901 . It fixes the `Makefile` to include the env variables needed in scope, and adds new sections to the [CONTRIBUTING](https://github.com/auth0/auth0-cli/blob/main/CONTRIBUTING.md) document, to make it more easy to troubleshoot or setup.

- Added explicit calls to the run script with the environment variable
- Added the `GOBIN` to the path for sake of the script
- Added clear instructions to CONTRIBUTING.md
- Unified `AUTH0_` environmental variable naming conventions

### 📚 References

Fixes issue #901 

### 🔬 Testing

```shell
make test-integration
```

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,6 +4,24 @@
 2. Clone this repo: `git clone git@github.com:auth0/auth0-cli`
 3. `make test` - ensure everything works correctly. Should see things pass.
 
+### Adjusting your Development Environment (Optional)
+
+To pass the integration tests, you must have the `AUTH0_CLI_CLIENT_DOMAIN`, `AUTH0_CLI_CLIENT_ID` and `AUTH0_CLI_CLIENT_SECRET` environment variable set. To get these values, you can:
+
+1. Install [jq](https://jqlang.github.io/jq/)
```
Comment (m3talsmith) [inline CONTRIBUTING.md:11]: An important dependency for the integration tests.

**Comment by m3talsmith** [comment]
@sergiught I made the environment variables uniform in my last commit. Currently the integration test results stand locally as:

Count: 243, Failed: 151, Skipped: 0

I'd appreciate if someone verify the results please.

**Comment by sergiught** [comment]
Thanks @m3talsmith 👋🏻 I'll take a look shortly.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,6 +4,24 @@
...
+3. Use the resulting **Client Secret** values
+
+#### Using the Bash or Zsh shell
+
+You can set these variables in your `~/.bashrc` or `~/.zshrc` (replace the values with your own):
+
+```shell
+export AUTH0_DOMAIN="yourappdomain"
+export AUTH0_CLIENT_ID="yourclientid"
+export AUTH0_CLIENT_SECRET="yourcedentialsecret"
```
Comment (willvedd) [inline CONTRIBUTING.md:22]: **Suggestion:** Could use placeholder values that better reflects the actual values:

```suggestion
export AUTH0_DOMAIN="travel0.us.auth0.com"
export AUTH0_CLIENT_ID="tUIvPH7g2ykVm4lGriYEQ6BKV3je24Ka"
export AUTH0_CLIENT_SECRET="XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
```

**Review by m3talsmith — COMMENTED** [review]
🚀

**Review by willvedd — APPROVED** [review]

**Review by Widcket — APPROVED** [review]

**Pull request merged**